<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Converter</class>
 <widget class="QWidget" name="Converter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Converter</string>
  </property>
  <widget class="QPushButton" name="binButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>bin</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="console">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>561</width>
     <height>521</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="numPanel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>787480576</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="multiplier">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bitButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>bit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hexButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>hex</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioBigButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Big Endian</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioLittleButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>99</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Little Endian</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mulButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>151</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>mul</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
